docs/config.ini

```diff
@@ -23,7 +23,8 @@ header_links = Stable Version Documentation, https://autogluon.mxnet.io/stable/i
                API, ###_PLACEHOLDER_WEB_CONTENT_ROOT_###/api/index.html, fas fa-book,
                Installation, ###_PLACEHOLDER_WEB_CONTENT_ROOT_###/install.html, fas fa-download,
                Tutorials, ###_PLACEHOLDER_WEB_CONTENT_ROOT_###/tutorials/index.html, fas fa-external-link-alt,
-               Github, https://github.com/awslabs/autogluon, fab fa-github
+               Github, https://github.com/awslabs/autogluon, fab fa-github,
+               Other Versions Documentation, ###_PLACEHOLDER_WEB_CONTENT_ROOT_###/versions.html, fas fa-book
```

Comment:
This should just link directly to /dev , otherwise old versions docs won't be able to link to newer version docs

Reply:
Updated.